tests/conftest.py

```diff
@@ -2,16 +2,16 @@
 import os
 
 
-token_openai = os.getenv("OPENAI_API_KEY")
+token_openai = os.getenv("PARSER_OPENAI_API_KEY")
```

Comment:
I think we could do some refactoring to take the environment key out of testing.

Reply:
this was to allow you to have it in your dev environment but not use it on the tests. any better idea?

Reply:
I'm gonna merge this PR as this is not a related topic, but please create an issue to improve it later